docs/Configuration.rst

```diff
@@ -18,40 +18,40 @@ Variables
 
 .. glossary::
 
-  **skip_migration_check**
-    **Default:** ``false``
+  **association_model_namespace**
```

Comment:
I took a moment to rearrange the Configuration::Variables section of the docs to be in alphabetical order